utils/node_local_test.py

```diff
@@ -1854,6 +1854,28 @@ def check_dfs_tool_output(output, oclass, csize):
     return True
 
 
+def check_dir_attr(output, oclass, dir_oclass, file_oclass, csize):
```

Comment:
The other option would be to change the calling code to pass the run_daos_cmd() the json=True option, in which case it would return a dict of the json contents.

Reply:
sure.. can keep that to an improvement patch for later